### PR TITLE
CLC-5965 Fix residency status display

### PR DESCRIPTION
**Pull request by raydavis**
https://jira.ets.berkeley.edu/jira/browse/CLC-5965
- Do not show residency status if no student record for current term
- Draw from official data source rather than munged version
- Reflect latest messaging suggestions from the Office of the Registrar
- Lower urgency of residency message while development continues

### CODE REVIEW

**Comment by raydavis** [comment]
Bernie had a good suggestion so I'm going to amend this PR.

File: app/models/campus_oracle/queries.rb [hunk 1]
```diff
@@ -168,6 +166,22 @@ def self.get_reg_status(person_id, term_yr, term_cd)
...
+        reg.admit_special_pgm_grp, reg.reg_special_pgm_cd, reg.cat_cd, reg.acad_blk_flag, reg.admin_blk_flag, reg.fin_blk_flag,
+        reg.tot_enroll_unit, reg.new_trfr_flag, reg.term_yr, reg.term_cd
+      from calcentral_person_info_vw pi, calcentral_student_term_vw reg where
+        reg.ldap_uid = #{person_id.to_i} and reg.ldap_uid = pi.ldap_uid
+      order by reg.term_yr desc, reg.term_cd desc
+        SQL
+        result = connection.select_all(sql)
+      }
+      stringify_ints! result
+    end
```
Comment (paulkerschen) [inline app/models/campus_oracle/queries.rb]: @raydavis - For my own edification: is this very useful-looking `get_student_term_info` query not yet being used in the codebase? Foundation work for more informative multi-term status messaging?

Comment (raydavis) [inline app/models/campus_oracle/queries.rb]: Utility method for this PR; foundation work for future. I'd be glad to amend the PR with a comment to that effect (I actually started typing one & then thought, "Nah, that's just distracting...")

**Comment by paulkerschen** [comment]
A great gain in clarity, @raydavis.

**Pull request merged**